dev/app/app.js

```diff
@@ -4,16 +4,12 @@ var angular = require('angular');
 require('angular-route');
 require('angular-sanitize');
 require('angularfire');
-/**
- * @ngdoc overview
- * @name freeCodeLondonersCampApp
- * @description
- * # freeCodeLondonersCampApp
- *
- * Main module of the application.
- */
 
-var dataSource = (require('./app.config')) ? require('./shared/services/local') : require('./shared/services/firebase');
+require('./assets/sass/main.scss');
```

Comment:
I don't think so. Why, does this cause problems?

I know in React you can use it like this:

```javascript
import styles from 'path/to/file.scss';
```
and within the jsx you can set the `className` attribute to `styles.classnamehere`

is that what you're looking for?